src/plone/restapi/services/content/add.py

```diff
@@ -1,13 +1,14 @@
 from Acquisition import aq_base
 from Acquisition.interfaces import IAcquirer
+from plone.app.multilingual.interfaces import IPloneAppMultilingualInstalled
```

Comment:
I think the `plone.app.multilingual` import should be still in a try/except ImportError block, but maybe I am wrong.

Reply:
No, plone.app.multilingual is included by default in Plone 5.2 and AFAIK there is no need to do a try: except: import.

Reply:
Exactly: https://github.com/plone/Products.CMFPlone/blob/master/setup.py#L59